### PR TITLE
[python3] Fix string no attribute error

### PR DESCRIPTION
**Pull request by kylazhang**
Change to str

Signed-off-by: Kylazhang <weizhan@redhat.com>

### CODE REVIEW

File: virttest/asset.py [hunk 1]
```diff
@@ -126,7 +126,7 @@ def check(self, section):
         options = self.parser.items(section)
         for i in range(options.__len__()):
             param = options[i][0]
-            aux = string.split(param, '.')
+            aux = str.split(param, '.')
```
Comment (sathnaga) [inline virttest/asset.py]: can we remove `string` import

File: virttest/element_tree.py [hunk 2]
```diff
@@ -852,7 +852,7 @@ def fixtag(tag, namespaces):
     # tag and namespace declaration, if any
```
Comment (sathnaga) [inline virttest/element_tree.py]:
```
import string


try:
    REPLACE = string.replace
except AttributeError:
    REPLACE = str.replace
```

we can remove this `string` import aswell and make `REPLACE=str.replace`

**Comment by sathnaga** [comment]
```
 python
Python 2.7.15 (default, May  2 2018, 14:12:52) 
[GCC 8.0.1 20180324 (Red Hat 8.0.1-0.20)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> params='sdskj.diude/lsadsjac.hsysnm8jsjsl.ksisls'
>>> str.split(params,'.')
['sdskj', 'diude/lsadsjac', 'hsysnm8jsjsl', 'ksisls']
>>> str.replace(params,'.',":")
'sdskj:diude/lsadsjac:hsysnm8jsjsl:ksisls'

-----
$ python3
Python 3.6.5 (default, Mar 29 2018, 18:20:46) 
[GCC 8.0.1 20180317 (Red Hat 8.0.1-0.19)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> params='sdskj.diude/lsadsjac.hsysnm8jsjsl.ksisls'
>>> import string
>>> str.split(params,'.')
['sdskj', 'diude/lsadsjac', 'hsysnm8jsjsl', 'ksisls']
>>> str.replace(params,'.',":")
'sdskj:diude/lsadsjac:hsysnm8jsjsl:ksisls'
```

File: virttest/element_tree.py (hunk 2 above)
Comment (kylazhang) [reply virttest/element_tree.py]: ok, thanks, I will remove them.

**Comment by kylazhang** [comment]
@sathnaga Already resolved, thanks

**Pull request merged**